modules/openconfig_gnmi/0.13.0/MODULE.bazel

```diff
@@ -0,0 +1,35 @@
+module(
+    name = "openconfig_gnmi",
+    version = "0.13.0",
+)
+
+bazel_dep(name = "gazelle", version = "0.41.0", repo_name = "bazel_gazelle")
+bazel_dep(name = "grpc", version = "1.69.0", repo_name = "com_github_grpc_grpc")
+bazel_dep(name = "protobuf", version = "29.3", repo_name = "com_google_protobuf")
+bazel_dep(name = "rules_go", version = "0.51.0", repo_name = "io_bazel_rules_go")
+
+go_sdk = use_extension("@io_bazel_rules_go//go:extensions.bzl", "go_sdk")
+go_sdk.download(version = "1.23.4")
+
+go_deps = use_extension("@bazel_gazelle//:extensions.bzl", "go_deps")
+go_deps.from_file(go_mod = "//:go.mod")
+use_repo(go_deps, "com_github_cenkalti_backoff_v4", "com_github_golang_glog", "com_github_google_go_cmp", "com_github_kylelemons_godebug", "com_github_openconfig_grpctunnel", "com_github_openconfig_ygot", "com_github_protocolbuffers_txtpbfmt", "org_bitbucket_creachadair_stringset", "org_golang_google_grpc", "org_golang_google_grpc_cmd_protoc_gen_go_grpc", "org_golang_google_protobuf", "org_golang_x_crypto", "org_golang_x_net")
+
+go_deps_dev = use_extension("@bazel_gazelle//:extensions.bzl", "go_deps", dev_dependency = True)
+go_deps_dev.gazelle_override(
+    directives = [
+        "gazelle:proto disable",  # keep
+        "gazelle:resolve go github.com/openconfig/gnmi/errlist @openconfig_gnmi//errlist:errlist",  # keep
+    ],
+    path = "github.com/openconfig/grpctunnel",
+)
+go_deps_dev.gazelle_override(
+    directives = [
+        "gazelle:resolve go github.com/openconfig/gnmi/proto/gnmi @openconfig_gnmi//proto/gnmi:gnmi",  # keep
+        "gazelle:resolve go github.com/openconfig/gnmi/errlist @openconfig_gnmi//errlist:errlist",  # keep
+        "gazelle:resolve go github.com/openconfig/gnmi/value @openconfig_gnmi//value:value",  # keep
+    ],
+    path = "github.com/openconfig/ygot",
+)
+
+inject_repo(go_deps, "openconfig_gnmi")
```

Comment:
`go_deps` is not a dev dependency, but `inject_repo` only works if the module is a root module, is this expected?

https://bazel.build/rules/lib/globals/module#inject_repo

Reply:
I created https://github.com/bazel-contrib/bazel-gazelle/issues/2022 because I can not find a great solution to this general problem. This is a bit of a hack to allow this to build when it is a root module. I don't think there is a difference if we inject into go_deps vs go_deps_dev; both seem to have the same result. Ideally I would like go_deps to include this path so this was not required to begin with.

This problem only shows up when the module is the root module, so the fact that this is ignored for others is great.